tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -196,9 +198,19 @@ object SQLPlanParser extends Logging {
       }
       val stagesInNode = getStagesInSQLNode(node, app)
       val supported = execInfos.isSupported && !ds && !containsUDF
+      // add Exec to the unsupported list if the speedup factor is 1 and all underlying
+      // expressions are supported.
+      val unSupportedExecs = if (!execInfos.isSupported && execInfos.speedupFactor == 1
+        && execInfos.unsupportedExprs.length == 0 ) {
+        execInfos.exec
+      } else {
+        ""
+      }
+      val unSupportedExprs = execInfos.unsupportedExprs
```

Comment:
unless it throws off the spacing no need for the extra val here just use execInfos.unsupportedExprs below

Reply:
done